salt/modules/localemod.py

```diff
@@ -315,23 +316,19 @@ def gen_locale(locale, **kwargs):
             append_if_not_found=True
         )
 
-    if salt.utils.which("locale-gen") is not None:
+    if salt.utils.which('locale-gen') is not None:
```

Comment:
Not sure if `is not None` is needed, although `salt.utils.which` has an explicit `return None`, which can be actually removed. It either returns a path or nothing.